debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/StartedMongoDbSinkTask.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import static io.debezium.connector.mongodb.sink.MongoDbSinkConnectorTask.LOGGER;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.OptionalLong;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.bson.BsonDocument;
+
+import com.mongodb.MongoBulkWriteException;
+import com.mongodb.MongoNamespace;
+import com.mongodb.bulk.BulkWriteResult;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.model.BulkWriteOptions;
+import com.mongodb.client.model.WriteModel;
+
+import io.debezium.dlq.ErrorReporter;
+
+final class StartedMongoDbSinkTask implements AutoCloseable {
```

Comment:
Is it possible to use `io.debezium.pipeline.sink.spi.ChangeEventSink`?
I can envision then `BaseSinkTask` that will manage the sink connector/task lifecycle.

Reply:
Also we should intorduce `BaseSinkConnector` even if it is empty at the start.

Reply:
Possibly yes, I will log a Jira for it. Looking at my upcoming PTO I want to merge it like that and refactor it with an extra task later. is this fine for you @jpechane ?

Reply:
filed https://issues.redhat.com/browse/DBZ-7996